### DISCUSSION TITLE
FontAwesome battery icons

### DISCUSSION THREAD

**Discussion by ghost** [polybar/polybar#2850] [Q&A]
Hey, I wanna get the charging and discharging animations in the battery module, and for that I use FontAwesome Icons.
I already added the FontAwesome Regular font.
For the Icons I tried the CSS Format <c...  and the fa-fa-battery-0 Format
They both dont seem to work, my polybar doesent show when I try these options.

**Comment by patrick96** [comment]
You need to copy-paste the icons as text characters, that's the only thing polybar understands. The font awesome website offers a way to do this (there it is called "Copy Glyph"). Another good tool is `gucharmap`. See also the [wiki](https://github.com/polybar/polybar/wiki/Fonts#gnome-character-map)
